### PR TITLE
Add a section on release wiki about pytorchbot cherry-pick command

### PR DESCRIPTION
**Pull request by huydhn**
I add a section about the new `pytorchbot cherry-pick` command in the release wiki so that more people know about it

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -232,6 +232,31 @@ Please also make sure to add milestone target to the PR/issue, especially if it
 
 **NOTE**: The cherry pick process is not an invitation to add new features, it is mainly there to fix regressions
 
+#### How to do Cherry Picking
+
+You can now use `pytorchbot` to cherry pick a PR that has been committed
+to the main branch using `@pytorchbot cherry-pick` command as follows.
```
Comment (atalman) [inline RELEASE.md:238]: Please list all the repos this feature is enabled in.

**Review by atalman — APPROVED** [review]

**Comment by atalman** [comment]
Please resolve lint

File: RELEASE.md (hunk 1 above)
Comment (huydhn) [reply RELEASE.md:238]: Good point.  As the bot is only available in PyTorch atm, this implies that we only have this one PyTorch.  Let me make that point clearer.

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**